.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,92 @@
+name: Build, test and package esque
+on:
+  push:
+    branches:
+      - 61-installation-and-packaging
+  pull_request:
+    branches:
+      - 61-installation-and-packaging
+
+jobs:
+  test-style:
+    name: Code style check on ${{ matrix.os }} with Python ${{ matrix.python }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ['3.x', '3.6', '3.7', '3.8', 'pypy3' ]
+        os: ['ubuntu-latest']
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python }}
+          architecture: x64
+        run: |
+          pre-commit install --install-hooks
+      - name: Run pre-commit hooks
+        run: pre-commit run
+  test-integration:
+    name: Integration test with Python ${{ matrix.python }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ['3.x', '3.6', '3.7', '3.8', 'pypy3' ]
+      max-parallel: 1
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup python and install packages
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python }}
+          architecture: x64
+        run: |
+          sudo apt-get install -y python3-setuptools python3-virtualenv python3-pip virtualenv python3-venv
+          pip3 install pre-commit
+      - name: Install and configure poetry
+        run: |
+          curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python3
+          export PATH=$PATH:$HOME/.poetry/bin
+          source $HOME/.poetry/env
+          virtualenv venv --python=python3
+          $HOME/.poetry/bin/poetry self:update --preview
+      - name: Install esque requirements
+        run: |
+          source ./venv/bin/activate
+          $HOME/.poetry/bin/poetry install -vvv
+      - name: Setup default esque configuration
```

Comment:
You don't need this step, esque test suite will use its own configuration from `tests/test_configs/integration_test_config.yaml`